doc/source/admin/conda_faq.rst

```diff
@@ -18,25 +18,24 @@ Not only do Conda packages make tool dependencies more reliable and
 stable, they are also easier to test and faster to develop than the
 traditional Tool Shed package recipes.
 
-Conda is a package manager like apt-get, yum, pip, brew or guix. We
-don't want to argue about the relative merits of various package
-managers here, in fact Galaxy supports multiple package managers and we
-welcome community contributions (such as implementing a Guix package
-manager or enhancing the existing brew support to bring it on par with
-Conda).
-
-As a community, we have decided that Conda is the one that best fulfills
-our needs. The following are some of the crucial Conda features that led
+Conda is a package manager like ``apt-get``, ``yum``, ``pip``, ``brew`` or
+``guix``. We don't want to argue about the relative merits of various package
+managers here, in fact Galaxy supports multiple package managers and we welcome
+community contributions (such as implementing a Guix package manager or
+enhancing the existing brew support to bring it on par with Conda).
+
+As a community, we have decided that Conda is the one that best fulfills the
+project's needs. The following are some of the crucial Conda features that led
```

Comment:
I'll update it; earlier formulation seemed read a bit selfishly despite the initial community involvement...